docs/src/prototypes/customToolbar/CustomToolbar.tsx

```diff
@@ -0,0 +1,230 @@
+import * as _ from 'lodash'
+import * as React from 'react'
+import {
+  Button,
+  Text,
+  Toolbar,
+  ToolbarItemProps,
+  ToolbarCustomItemProps,
+  Status,
+  ToolbarItemShorthandKinds,
+} from '@stardust-ui/react'
+
+export interface CustomToolbarProps {
+  layout?: 'standard' | 'desktop-share' | 'powerpoint-presenter'
+
+  isRecording?: boolean
+
+  cameraActive?: boolean
+  onCameraChange?: (state: boolean) => void
+
+  micActive?: boolean
+  onMicChange?: (state: boolean) => void
+
+  screenShareActive?: boolean
+  onScreenShareChange?: (state: boolean) => void
+
+  sidebarSelected: false | 'chat' | 'participant-add'
+  onSidebarChange?: (state: false | 'chat' | 'participant-add') => void
+
+  chatHasDot?: boolean
+
+  pptSlide?: string
+  onPptPrevClick?: () => void
+  onPptNextClick?: () => void
+
+  onEndCallClick?: () => void
+}
+
+type CustomToolbarItem = (ToolbarItemProps | ToolbarCustomItemProps) & {
+  key: string
+  kind?: ToolbarItemShorthandKinds
+}
+type CustomToolbarLayout = (props: CustomToolbarProps) => CustomToolbarItem[]
+
+const commonLayout: CustomToolbarLayout = props =>
+  [
+    props.isRecording && {
+      key: 'recording',
+      kind: 'custom' as ToolbarItemShorthandKinds,
+      focusable: true,
+      content: (
+        <Status
+          state="error"
```

Comment:
Hmm we can strange name for error... Maybe danger would be better choice.. :\

Reply:
This PR should not change any library code